test/client/modules/login/index.spec.js

```diff
@@ -0,0 +1,105 @@
+describe('LoginCtrl', function() {
+
+    var $controller, scope, uibModalInstance, items, sendLog, loginResult, httpBackend;
+    var controller;
+
+    beforeEach(angular.mock.module("tanks.login"));
+
+    // describe('Testing sendLog service', function() {
```

Comment:
so ask here, not sure I will be able to attend class tomorrow. Maybe after 2 pm... not sure.